spec/requests/api/merge_requests_spec.rb

```diff
@@ -364,6 +385,15 @@
       json_response.first['author']['id'].should == user.id
     end
 
+    it "should return comments of merge_request by iid" do
+      get api("/projects/#{project.id}/merge_request/#{merge_request.iid}/comments", user)
+      response.status.should == 200
+      json_response.should be_an Array
+      json_response.length.should == 1
+      json_response.first['note'].should == "a comment on a MR"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.